src/main/java/de/tum/in/www1/jenkins/notifications/model/ObjectFactory.java

```diff
@@ -0,0 +1,22 @@
+package de.tum.in.www1.jenkins.notifications.model;
+
+import javax.xml.bind.annotation.XmlRegistry;
+
+/**
+ * Based on <a href="https://github.com/jenkinsci/performance-signature-dynatrace-plugin/pull/81/files">an issue of
+ * another Jenkins Plugin</a> I've created this object factory to ensure that JAXB works in JDK11.
+ *
+ * @author Dominik Fuchss
+ *
+ */
+@XmlRegistry
+public class ObjectFactory {
+
+    public ObjectFactory() {
+        // NOP
```

Comment:
```suggestion
        // The empty constructor is needed explicitly so that it can be accessed with reflection for JAXB.
```
I would say it is fine to ignore Cadacy in that case if some comment explains why the empty constructor is needed.
(I have not tested if it works without the explicit constructor.)

Reply:
Sounds good. I'll apply that.